content/compose/environment-variables/env-file.md

```diff
@@ -1,64 +0,0 @@
----
-description: Understand the accepted syntax for declaring environment variables with an environment file
-keywords: fig, composition, compose, docker, orchestration, environment, env file
-title: Syntax for environment files in Docker Compose
-aliases:
-- /compose/env-file/
```

Comment:
Don't forget to add a redirect for this page and to copy this alias as well (there's probably many references to this page)